### PR TITLE
Wno-error=sign-compare 4 fuzztest on GLIBC>2.34-8

### PR DESCRIPTION
**Pull request by aferr**
This is part of a broader quest to make Raksha buildable on my cloudtop again

### CODE REVIEW

**Comment by aferr** [comment]
The second commit should help make this work even if google-fuzztest folks don't agree with my issue or take my change:
https://github.com/google/fuzztest/issues/64
https://github.com/google/fuzztest/pull/65

**Comment by aferr** [comment]
@bgogul Is this one OK ?

File: .bazelrc [hunk 1]
```diff
@@ -10,6 +10,11 @@ build --cxxopt='-Werror' --cxxopt='-Wall'
 build --features=external_include_paths
 build --cxxopt='-Wno-deprecated-declarations'
 
+# These are needed on systems with GLIBC > 2.34-8 to build google-fuzztest
```
Comment (bgogul) [inline .bazelrc:13]: You might want to move it below where there is already some exceptions related to fuzztest?

You will also need to add an equivalent `--host_cxxopt`

Comment (aferr) [reply .bazelrc]: fixed

**Review by bgogul — APPROVED** [review]

**Pull request merged**